<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StopWatch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPause">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLane1Lap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lane 1 Lap</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twLapTimes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLane2Lap">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lane 2 Lap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSQLTest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
